pom.xml

```diff
@@ -86,7 +86,7 @@
     <dependency>
       <groupId>org.folio</groupId>
       <artifactId>edge-common</artifactId>
-      <version>3.0.0</version>
+      <version>4.2.0</version>
     </dependency>
```

Comment:
Found it was necessary to upgrade the edge-common dependency as well.  the older version does not seem to be compatible with vertx 4.